lib/measured/conversion_builder.rb

```diff
@@ -0,0 +1,35 @@
+class Measured::ConversionBuilder
+  def initialize
+    @base_unit = nil
+    @units = []
+    @case_sensitive = false
+  end
+
+  def base(unit_name, aliases: [])
+    raise Measured::UnitError, "Setting #{unit_name} as base unit, but already defined as #{@base_unit}." if @base_unit
+    @base_unit = Measured::Unit.new(unit_name, aliases: aliases)
+  end
+
+  def unit(unit_name, aliases: [], value:)
+    @units << Measured::Unit.new(unit_name, aliases: aliases, value: value)
```

Comment:
Shovel operator returns the resulting collection.

If we're defining DSL operations we should be opinionated about their returns, even if we're ignoring them.

Pick a pattern and either return the added unit object, or always `nil`.